### DISCUSSION TITLE
flomo + rss：在 RSS 阅读器中使用API 同步批注、笔记到 flomo

### DISCUSSION THREAD

**Discussion by maoruibin** [flomoapp/3rd-party-tools#63] [分享创造]
![](https://gitee.com/maoruibin/img/raw/master/2022/01/20/20220120141548960.png)

DongReader 是一个 Android RSS 阅读器，主打简洁实用，最近的版本加入了批注功能，用来记录阅读文章时的灵感想法，由于 flomo 提供了 API，所以也支持了自动同步到 flomo 笔记中。

DongReader 下载地址：http://d.firim.top/gulz

更多详细介绍：https://gudong.site/2022/01/20/rss-flomo.html

**Comment by eatmore** [comment]
现在链接失效了
